### ISSUE TITLE
feed order

### ISSUE THREAD

**Issue by changho3995** [Ashinch/ReadYou#120]
**1. Environment**

-   Device: [Pixel 6]
-   OS: [Android 12.1]
-   APP: [0.7.2]

**2. Describe the feature request**

(Want you've done some research before now)

I wish it was possible to sort the content(feed) list in oldest order...

**Comment by pc00per** [comment]
Currently it's always been newest first. 
Also, I feel like the feeds of main page also might need this sorting, since you can't find a feed in a group when they're a lot, caz they're not in any alphabetical order.

**Comment by changho3995** [comment]
It would be nice if the article list could also be sorted by oldest.

**Comment by mosource21** [comment]
Yes a sort oldest first option would be a great addition.  If you do not read your feeds all the time and articles mount up and if the articles in the feed can rely/follow on from each other it is important to read them oldest first.  I scroll to the bottom and read upwards but you always catch glimpses of articles released later and it can spoil the flow and it is odd having to scroll up the list rather than down.  Thanks

**Comment by Tywele** [comment]
I was very surprised to find out that you can't change the sorting of articles to "show oldest first" in this app. It seems like such a basic feature. So this is me agreeing with the others that this should be implemented.

**Comment by dsh2dsh** [comment]
Same here. I'd prefer oldest first too.

**Comment by mgnsk** [comment]
I scroll through thousands of headlines throughout the day in another reader app that supports this. I'm using the `Mark as read on scroll` feature so that when I come back I can continue where I left off. A `scroll to oldest news` option wouldn't work as it would either mark everything as read or lose the scroll position.

The oldest first order is the only thing stopping me from using this app.

**Comment by garret** [comment]
> I scroll through thousands of headlines throughout the day in another reader app that supports this. I'm using the `Mark as read on scroll` feature so that when I come back I can continue where I left off. A `scroll to oldest news` option wouldn't work as it would either mark everything as read or lose the scroll position.
> 
> The oldest first order is the only thing stopping me from using this app.

I am in the exact same situation.

**Comment by JunkFood02** [comment]
Hi everyone! You might want to preview the changes in #992 with the [ci artifact here](https://github.com/Ashinch/ReadYou/actions/runs/14431609970?pr=992). Don't forget to backup your data before installing unstable dev builds 🤞

**Comment by dsh2dsh** [comment]
@JunkFood02 thank you! Looks like it works for me. I compared list of articles in Read You with Miniflux and the first/last articles are same in Read You (with new option) and Miniflux.
